auv_control_demos/README.md

```diff
@@ -0,0 +1,321 @@
+# AUV Control Demos
+
+This package contains examples that demonstrate how to use the controllers in
+`auv_controllers`. There are two examples. The first shows how to launch just a
+single controller on its own. The second example shows how to chain the
```

Comment:
Maybe revise to something like: "The second example demonstrates how to chain the controllers together to construct a cascading control architecture."

Reply:
done